src/main/kotlin/pt/up/fe/ni/website/backend/service/ActivityService.kt

```diff
@@ -32,4 +36,59 @@ abstract class ActivityService<T : Activity>(
         activity.teamMembers.removeIf { it.id == idAccount }
         return repository.save(activity)
     }
+
+    fun addHallOfFameMemberById(idActivity: Long, idAccount: Long): T {
```

Comment:
for the same reason as above, all of the logic here would be ported into ProjectService